azerothcore/azerothcore-wotlk
# Fix(Creature/AI): Mirveda's summoned creatures should despawn out of …

…combat



## Changes Proposed:
-  Change summon type and reduce duration to 15 seconds ooc.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11901
- Closes https://github.com/chromiecraft/chromiecraft/issues/3559

## SOURCE:


## Tests Performed:

- tested


## How to Test the Changes:


1.Start the quest chain with [The Wayward Apprentice](https://tbc.wowhead.com/quest=9254/the-wayward-apprentice) in Eversong Woods
2. Come to Apprentice Mirveda and complete [Corrupted Soil quest](https://classic.wowhead.com/quest=8487/corrupted-soil)
3. Take the quest [Unexpected Results](https://classic.wowhead.com/quest=8488/unexpected-results)
4. Fail the quest by jumping in front and rounding mobs with Arcane Torrent, have both you and Mirveda killed.
5. See that the mobs despawn out of combat after 15 seconds.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
